release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1819,6 +1819,14 @@ As a workaround, if you have an installation configuration file, update it with
 
 * When you deploy a cluster with a Network Type of `OVNKubernetes` on {ibmpowerProductName}, compute nodes may reboot because of a kernel stack overflow. As a workaround, you can deploy the cluster with a Network Type of `OpenShiftSDN`. (link:https://issues.redhat.com/browse/RHEL-3901[*RHEL-3901*])
 
+* The following known issue applies to users who upgraded their {product-title} deployment to an early access version of {product-version} using release candidate 3 or 4: 
++
+After the introduction of the node identify feature, some pods that were running as root are updated to run unprivileged. For users who updated to an early access version of {product-title} {product-version}, attempting to upgrade to the official {product-version} might not progress with the Network Operator reporting the following state persistently: `DaemonSet "/openshift-network-node-identity/network-node-identity" update is rolling`. 
```

Comment:
>  might not progress with the Network Operator reporting

I'm not clear on whether what _the Network Operating reporting_ is good or bad. Is the update supposed to progress with this state, or does the state indicate something is wrong?

Reply:
I believe we update cluster and upgrade Operators. 
```suggestion
After the introduction of the node identify feature, some pods that were running as root are updated to run unprivileged. For users who updated to an early access version of {product-title} {product-version}, attempting to update to the official {product-version} might not progress with the Network Operator reporting the following state persistently: `DaemonSet "/openshift-network-node-identity/network-node-identity" update is rolling`. 
```